### ISSUE TITLE
Proj0036: False positive on actually hidden files

### ISSUE THREAD

**Issue by CptWesley** [dotnet-project-file-analyzers/dotnet-project-file-analyzers#555]
```xml
  <PropertyGroup>
    <DocumentationFile>foo.xml</DocumentationFile>
  </PropertyGroup>

  <ItemGroup>
    <None Remove="foo.xml" />
  </ItemGroup>
```

Causes the documentation to be written to the project root, but excluded from the IDE view, yet `Proj0036` triggers.

**Comment by Corniel** [comment]
This can also be achieved with `<None Update="foo.xml" Visible="false" />`, but I guess Remove is preferred as being shorter.
